### PR TITLE
SCR model change + Support temporal property types

### PR DESCRIPTION
**Pull request by vyacheslav-pushkin**

### CODE REVIEW

**Comment by vadimbasko** [comment]
scripts/model/projectModel-scr.json 

It's better to format this file using IDEA before commit to git. Sometimes it helpful to analyze it for tracking changes.

**Comment by vadimbasko** [comment]
I think no need to support hsql further. I think we can remove db/init/hsql folder.

File: README.md [hunk 1]
```diff
@@ -498,16 +498,41 @@ As soon as we decorate a class or a function component as [observer](https://mob
...
+export interface CubaAppConfig {
+  dataTransferFormats?: Partial<Record<PropertyType, string>>;
+  displayFormats?: Partial<Record<PropertyType, string>>;
+}
+```
+
+**dataTransferFormats** can be used to override the default formats used to (de)serialize the data transferred by REST API.
+
+**displayFormats** can be used to override the formats used for data presentation. 
+
```
Comment (vadimbasko) [inline README.md]: Is there somewhere in docs\code list of formats that user can be overrides ad display\dataTransfer? If no, I think it make sense to add it, or may be add link to CUBA docs with formats description.

File: packages/cuba-react-ui/src/ui/EntityProperty.tsx [hunk 2]
```diff
@@ -74,3 +82,13 @@ function formatValue(value: any): string {
   }
   return JSON.stringify(value);
 }
+
+export function toDisplayValue(value: any, propertyInfo: MetaPropertyInfo) {
+  if (value != null && isTemporalProperty(propertyInfo)) {
```
Comment (vadimbasko) [inline packages/cuba-react-ui/src/ui/EntityProperty.tsx]: The test will not be superfluous here.

File: packages/cuba-react-ui/src/ui/EntityProperty.tsx [hunk 3]
```diff
@@ -74,3 +82,13 @@ function formatValue(value: any): string {
   }
   return JSON.stringify(value);
 }
+
+export function toDisplayValue(value: any, propertyInfo: MetaPropertyInfo) {
```
Comment (vadimbasko) [inline packages/cuba-react-ui/src/ui/EntityProperty.tsx]: The test will not be superfluous here.

File: packages/cuba-react-core/src/util/formats.ts [hunk 4]
```diff
@@ -1,3 +1,45 @@
...
+  dateTime: defaultDateTimeDisplayFormat,
+  localDate: defaultDateFormat,
+  localTime: defaultTimeFormat,
+  localDateTime: defaultDateTimeDisplayFormat,
+  offsetDateTime: offsetDateTimeDisplayFormat,
+  offsetTime: offsetTimeDisplayFormat
+};
+
+export function getDataTransferFormat(type: PropertyType): string | undefined {
+  const formats = {...dataTransferFormats, ...getCubaAppConfig()?.dataTransferFormats};
```
Comment (web-devel) [inline packages/cuba-react-core/src/util/formats.ts]: I'm not sure why we need to create the whole `formats` object each time here (and in `getDisplayFormat`, if we know what property type we are looking for?

**Pull request merged**